packages/components/src/Combobox/hooks/useMakeComboboxHandlers.ts

```diff
@@ -34,14 +34,6 @@ export function useMakeComboboxHandlers(
     setOpen(true);
   }, [setOpen]);
 
-  const toggleOpen = useCallback(() => {
```

Comment:
the only place this was used was in Trigger & Activator, both of which can't actually be interacted with while `open` is `false` making the first code path inaccessible.

if the first code path is not real, then we're left with `handleOpen` and we might as well use that and get rid of `toggleOpen`